source/config/configSpec.py

```diff
@@ -37,6 +37,7 @@
 	outputDevice = string(default=default)
 	autoLanguageSwitching = boolean(default=true)
 	autoDialectSwitching = boolean(default=false)
+	increaseSayAllCaretUpdates = boolean(default=false)

```

Comment:
Should this be a speech synth dependent setting? For example, users may have many synths, and  some might work better with this feature than others.

Reply:
@derekriemer wrote:

> Increase caret updates during say all sounds wordy to me. How about "Move cursor by word during say all"

Because while the caret is intended to move per word, the movement is also limited to once per half a second.

Reply:
Ugh, sorry for the comment above, it ended up at the wrong place.

Anyway, making this a per synth setting makes sense to me. I will do that.